exercises/set_intersection/python/tests.py

```diff
@@ -0,0 +1,21 @@
+import unittest
+from main import *
+
+def f1(n):
+    return n
+def f2(n):
+    return n ** s2
+def f3(n):
+    return 3-n
+
+class MyTestCase(unittest.TestCase):
```

Comment:
it is usually good practice to have a few tests with normal cases (like you have), along with tests of the possible edge cases (which you should add), for example, if one of the lists is empty.